cmd/start.go

```diff
@@ -221,6 +221,8 @@ func initializeClientInfo(
 		build.Revision,
 	)
 
+	registry.RegisterChainInfo(blockCounter)
```

Comment:
Can we rename it to `RegisterChainInfoSource` to keep it consistent with `RegisterConnectedpeersSource` and `RegisterClientInfoSource`? 

Let's also take an opportunity and add an empty line between the calls.

Reply:
Sure [0493b72](https://github.com/keep-network/keep-core/pull/3637/commits/0493b72b3f8e4c41684037eaa534beef7433474c)